modin/engines/ray/pandas_on_ray/frame/partition.py

```diff
@@ -136,19 +143,22 @@ def deserialize(obj):
         for func, kwargs in call_queue[:-1]:
             func = deserialize(func)
             kwargs = deserialize(kwargs)
-            oid_obj = func(oid_obj, **kwargs)
+            try:
+                partition = func(partition, **kwargs)
+            except ValueError:
+                partition = func(partition.copy(), **kwargs)
```

Comment:
What's strange is that copying is not working anymore for some reason, particularly here. It used to work, but here it is not. I changed a bit of the code to avoid any Arrow errors, but I find it strange that this used to fix the problem but no longer does.